.buildkite/pipeline.yml

```diff
@@ -1,5 +1,46 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
-steps:
-  - label: "Example test"
-    command: echo "Hello!"
+env:
```

Comment:
Missing to add in this PR the `.buildkite/post-checkout` hook.

That hook needs to be added in all repositories for now, so the builds triggered in Pull Requests run the commit resulting of merging the target branch with the head of the Pull Request.

Reply:
added